src/pages/ServicesTests.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from "react";
+
+import {
+  FetchResult,
+  WeatherResponseFormat,
+  ClimateDetails,
+} from "../services/services.inteface";
+
+import {
+  RequestByName,
+  RequestByLattAndLong,
+  climateCityDetails,
+} from "../services/api.requests";
+
+import { getWeatherImageUrl } from "../services/image.request";
+
+import "../styles/services.css";
+
+//TODO: EXCLUIR BUID E npm uninstall serve
+
+export const ServicesTestsPage: React.FC = () => {
+  const [text, setText] = useState("");
+  const [data, setData] = useState<FetchResult[]>([]);
+  const [weatherData, setWeatherData] = useState<WeatherResponseFormat[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  //get client's current localization
+  useEffect(() => {
+    async function getCurrentLocalization() {
+      const response = await RequestByLattAndLong();
+      if (response) {
+        handleListButton(response[0].woeid);
+      }
+    }
+    getCurrentLocalization();
+  }, []);
+
+  //get searched city
+  function handleClick() {
+    async function makeRequest() {
```

Comment:
Why create the makeRequest function inside the handleClick function? Why not just put the makeRequest content inside the handleClick?